<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrayScaleWidget</class>
 <widget class="QWidget" name="GrayScaleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>52</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GrayScaleWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Convert To Gray Scale</string>
   </property>
   <widget class="QLabel" name="labelRChannel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGRatio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.587</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGChannel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBRatio">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.114</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelChannelRatio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel Ratio</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBChannel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRRatio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.299</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvert">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
